projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```
@@ -2335,6 +2335,13 @@
         }
     }
 
+    %igx-grid__addrow-snackbar {
+        position: absolute;
+        z-index: 5;
+        left: 35%;
```

This property should be the interpolated version of the `$left` variable for RTL compatibility purposes.
```suggestion
        #{$left}: 35%;
```

Also, are you sure you want to use percentages as the unit to set the position of the snackbar? If yes, how did you come up with those numbers?